src/Registry/TypeRegistry.php

```diff
@@ -1157,7 +1157,12 @@ public function register_mutation( $mutation_name, $config ) {
 						// Translators: The placeholder is the name of the mutation
 						throw new Exception( sprintf( __( 'The resolver for the mutation %s is not callable', 'wp-graphql' ), $mutation_name ) );
 					}
-					$payload = call_user_func( $mutateAndGetPayload, $args['input'], $context, $info );
+
```

Comment:
@markkelnar nice! Are you able to get your code in the persisted queries plugin working with this change?